README.md

```diff
@@ -1,2 +1,162 @@
-# awesome-book-new
-Develope awesome book application with html and javascript
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+=======
+
+## Capstone-Project: Programming Competion-page
+
+
+[Loom Video Presentation]()
+>>>>>>> efb449574a89f51f74a8b2fc2fbc8a9489b5f399
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
Kindly remove this part of the README template. This is irrelevant to the project and is only for reference when you use the README template provided by the microverse.

Reply:
Fixed! 🔧